CITATION

```diff
@@ -0,0 +1,5 @@
+Please open
+
+    https://aspect.geodynamics.org/citing.html&src=citation
```

Comment:
This does not work for me (I get 404 Not Found).

Reply:
good catch!